grpc-examples/pom.xml

```diff
@@ -5,7 +5,7 @@
     <parent>
         <artifactId>stargate</artifactId>
         <groupId>io.stargate</groupId>
-        <version>1.0.43-SNAPSHOT</version>
+        <version>1.0.43</version>
```

Comment:
Should this also be 1.0.44-SNAPSHOT?

Reply:
Yea just wrote above, what's happening here?